src/GovUk.Education.ExploreEducationStatistics.Admin/appsettings.json

```diff
@@ -24,9 +24,6 @@
     "RefreshTokenExpiration": "Absolute"
   },
   "OpenIdConnect": {
-    "ClientId": "97d25f91-bd4b-4b82-843b-2583b8194800",
-    "ClientSecret": "C@R3I_k42?3=15W5-KzV=Nbpg.eBq/:4",
```

Comment:
Given that details are now firmly engraved into the Git history, are we able to rotate these credentials in our Hive AD once this change has gone in?

Reply:
Yeah, we'll be removing the App Registration altogether, so it'll no longer be a "thing".